pages/common/travis.md

```diff
@@ -0,0 +1,24 @@
+# travis
+
+> Command line client to interface with a Travis CI service.
+> More information: <https://github.com/travis-ci/travis.rb>.
+
+- Display the client version:
+
+`travis version`
+
+- Authenticate the CLI client against the server, using an authentication token:
+
+`travis login`
+
+- List repositories the user has certain permissions on:
+
+`travis repos`
+
+- Encrypt values for the .travis.yml:
```

Comment:
Perhaps this will read better?
```suggestion
- Encrypt values in `.travis.yml`:
```

Also, what's the token value for?

Reply:
I have committed your suggestion. The token value is the data to be encrypted. An example is: `travis encrypt SOMEVAR="secretvalue"` This is to store encrypted environment variables in Travis.